libs/wingc/src/type_check.rs

```diff
@@ -2988,6 +3030,15 @@ mod tests {
 		assert!(!Phase::Inflight.is_subtype_of(&Phase::Preflight));
 	}
 
+	fn make_function(params: Vec<TypeRef>, ret: TypeRef, phase: Phase) -> Type {
```

Comment:
probably need this to avoid a dead code warning
```suggestion
	#[cfg(test)]
	fn make_function(params: Vec<TypeRef>, ret: TypeRef, phase: Phase) -> Type {
```

Reply:
`#[cfg(test)]` is already applied to the module it's in (`mod tests`) so I haven't gotten any dead code warnings